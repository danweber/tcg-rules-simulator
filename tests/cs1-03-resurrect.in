VERSION:2
#\uff1cResurrection\uff1e (When this Monster with evolution cards is deleted, play this card without paying the cost.
FIELD1:Attacker,CS1-03,DW1-Van Sourceless,CS1-03
FIELD2:Tank,CS2-12,REST
GAME:P1 T2 MAIN 5
DUMP

ECHO: -- Attack into Tank, die, then resurrect by mandatory effect.
CMD:ATTACK Attacker Tank
DUMP
ECHO: -- Attack without sources, stay ded
CMD:ATTACK Sourceless Tank
DUMP:TRASH
