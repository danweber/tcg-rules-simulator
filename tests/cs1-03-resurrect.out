FIELD1:Attacker,CS1-Garbagecanlid,DW1-Vanillamon Sourceless,CS1-Garbagecanlid
FIELD2:Tank,CS2-Colossalsword,REST
GAME:P1 T2 MAIN 5
ECHO: -- Attack into Tank, die, then resurrect by mandatory effect.
FIELD1:Sourceless,CS1-Garbagecanlid I4,CS1-Garbagecanlid
FIELD2:Tank,CS2-Colossalsword,REST
GAME:P1 T2 MAIN 5
ECHO: -- Attack without sources, stay ded
FIELD1:I4,CS1-Garbagecanlid
FIELD2:Tank,CS2-Colossalsword,REST
TRASH1:DW1-Vanillamon CS1-Garbagecanlid
TRASH2:
GAME:P1 T2 MAIN 5
